### ISSUE TITLE
Request of a new group

### ISSUE THREAD

**Issue by MandyA** [MagneticParticleImaging/MDF#43]
For simulations it is necessary to have information about the phantom used. That is why it would be great to have an optional group phantom with a paramter data. I have to admit that currently this is the only parameter necessary. But I think it would be wise to have a specific group phantom rather than putting it in some other group. The only group where it might be possible is tracer, but I think having the phantom as tracer property would be misleading. I'm open for further suggestions.

**Comment by tknopp** [comment]
actually this is quite similar to what is contained in the reconstruction group, no? This is how it once was implemented in IMT framework, i.e. that phantom and reconstruction data have the same file format.

**Comment by MandyA** [comment]
> actually this is quite similar to what is contained in the reconstruction group, no? This is how it once was implemented in IMT framework, i.e. that phantom and reconstruction data have the same file format.

Yes, precisely - it still is implemented like that ;) So to use MDF with our complete simulation chain a phantom group would be nice. It would be a workaround to use reconstruction, if a new group would be too irretating for other MDF users.

**Comment by profix898** [comment]
No actual preference on this one, but I think we could factor phantom (geometry) and tracer (material) into one/two separate groups (as they are closely related anyway).

**Comment by tknopp** [comment]
This is an area where MDF is indeed not so well designed for. Our primary use case is to destribute measurment data. Feel free to make proposals how the MDF can improve in that area! best is to work on the tex file and make pull requests that can be discussed.
